modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/object-view/ViewBuilderListItem.tsx

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayButtonWithIcon} from '@clayui/button';
+import ClayList from '@clayui/list';
+import classNames from 'classnames';
+import React, {useContext, useRef} from 'react';
+import {useDrag, useDrop} from 'react-dnd';
+
+import ViewContext, {TYPES} from './context';
+import {TObjectViewColumn} from './types';
+
+import './ViewBuilderListItem.scss';
+
+interface Iprops {
+	index: number;
+	objectViewColumn: TObjectViewColumn;
+}
+
+const ViewBuilderListItem: React.FC<Iprops> = ({index, objectViewColumn}) => {
+	const [, dispatch] = useContext(ViewContext);
+
+	const {objectFieldName} = objectViewColumn;
+
+	const ref = useRef<any>();
```

Comment:
The more strict the better!
```suggestion
	const ref = useRef<HTMLLIElement>();
```